.github/workflows/codeql-workflow.yml

```diff
@@ -17,8 +17,6 @@ jobs:
     strategy: 
       matrix:
         include:
-          - SDK: "$NANOS_SDK"
```

Comment:
Is it on purpose?

Reply:
Yes, Nano S won't have future versions and the main branches are no longer compatible with Nano S starting from this version.

The last Nano S version is kept on its own `nanos` branch.